_episodes/07-testing.md

```diff
@@ -405,14 +405,13 @@ TDD has another benefit of never having false positives. If you ensure that your
 >     
 >     calculated_mass = molecool.calculate_molecular_mass(symbols)
 > 
->     actual_mass = molecool.atom_data.atom_weights['C'] + molecool.atom_data.atom_weights['H'] +\
->          molecool.atom_data.atom_weights['H'] + molecool.atom_data.atom_weights['H']+ molecool.atom_data.atom_weights['H']
->     
+>     actual_mass = sum(molecool.atom_data.atom_weights[atom] for atom in symbols)
```

Comment:
I thought this suggestion might reduce code repetition in case of changes in the future.